code/modules/mob/living/silicon/pai/software_modules.dm

```diff
@@ -632,6 +632,7 @@
 		user.add_language("Bubblish")
 		user.add_language("Orluum")
 		user.add_language("Clownish")
+		user.add_language("Sign Language")
```

Comment:
Does this allow borgs to speak this language?

Reply:
I'll double check that, but my intention was for it not to. I did a check to see if the person signinghas both hand "organs", but looking at it, it's only checking for ishuman() for that. I'll fix that and test further. 

I'm thinking I should probably put a check in for both hands being empty also while i'm at it.